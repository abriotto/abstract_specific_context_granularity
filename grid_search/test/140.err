  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  1%|#8                                                                                                                                                                                          | 1/99 [00:00<00:13,  7.22it/s]  3%|#####6                                                                                                                                                                                      | 3/99 [00:00<00:07, 13.12it/s]  6%|###########3                                                                                                                                                                                | 6/99 [00:00<00:06, 14.30it/s]  8%|###############1                                                                                                                                                                            | 8/99 [00:00<00:07, 12.57it/s] 10%|##################8                                                                                                                                                                        | 10/99 [00:00<00:08, 10.17it/s] 12%|######################6                                                                                                                                                                    | 12/99 [00:01<00:07, 11.14it/s] 14%|##########################4                                                                                                                                                                | 14/99 [00:01<00:08, 10.06it/s] 16%|##############################2                                                                                                                                                            | 16/99 [00:01<00:07, 11.37it/s] 18%|##################################                                                                                                                                                         | 18/99 [00:01<00:08,  9.05it/s] 20%|#####################################7                                                                                                                                                     | 20/99 [00:01<00:07, 10.43it/s] 22%|#########################################5                                                                                                                                                 | 22/99 [00:02<00:08,  9.58it/s] 24%|#############################################3                                                                                                                                             | 24/99 [00:02<00:08,  9.20it/s] 26%|#################################################1                                                                                                                                         | 26/99 [00:02<00:07, 10.29it/s] 28%|####################################################8                                                                                                                                      | 28/99 [00:02<00:07,  9.50it/s] 30%|########################################################6                                                                                                                                  | 30/99 [00:03<00:08,  7.81it/s] 31%|##########################################################5                                                                                                                                | 31/99 [00:03<00:09,  7.25it/s] 32%|############################################################4                                                                                                                              | 32/99 [00:03<00:09,  7.21it/s] 34%|################################################################2                                                                                                                          | 34/99 [00:03<00:08,  7.23it/s] 35%|##################################################################1                                                                                                                        | 35/99 [00:03<00:09,  7.07it/s] 36%|####################################################################                                                                                                                       | 36/99 [00:04<00:09,  6.57it/s] 38%|#######################################################################7                                                                                                                   | 38/99 [00:04<00:08,  7.07it/s] 40%|###########################################################################5                                                                                                               | 40/99 [00:04<00:06,  8.84it/s] 42%|###############################################################################3                                                                                                           | 42/99 [00:04<00:05, 10.14it/s] 44%|###################################################################################1                                                                                                       | 44/99 [00:04<00:05, 10.54it/s] 46%|######################################################################################8                                                                                                    | 46/99 [00:04<00:05, 10.56it/s] 48%|##########################################################################################6                                                                                                | 48/99 [00:05<00:06,  8.01it/s] 49%|############################################################################################5                                                                                              | 49/99 [00:05<00:06,  7.39it/s] 52%|################################################################################################3                                                                                          | 51/99 [00:05<00:06,  7.43it/s] 53%|##################################################################################################2                                                                                        | 52/99 [00:05<00:06,  7.04it/s] 55%|#####################################################################################################9                                                                                     | 54/99 [00:06<00:05,  8.33it/s] 57%|#########################################################################################################7                                                                                 | 56/99 [00:06<00:04,  9.18it/s] 58%|###########################################################################################################6                                                                               | 57/99 [00:06<00:05,  8.35it/s] 59%|#############################################################################################################5                                                                             | 58/99 [00:06<00:05,  7.79it/s] 61%|#################################################################################################################3                                                                         | 60/99 [00:06<00:04,  9.11it/s] 63%|#####################################################################################################################1                                                                     | 62/99 [00:06<00:03,  9.99it/s] 65%|########################################################################################################################8                                                                  | 64/99 [00:07<00:03,  9.87it/s] 67%|############################################################################################################################6                                                              | 66/99 [00:07<00:02, 11.27it/s] 69%|################################################################################################################################4                                                          | 68/99 [00:07<00:02, 12.49it/s] 71%|####################################################################################################################################2                                                      | 70/99 [00:07<00:03,  9.27it/s] 73%|########################################################################################################################################                                                   | 72/99 [00:07<00:02,  9.67it/s] 75%|###########################################################################################################################################7                                               | 74/99 [00:08<00:02,  9.30it/s] 77%|###############################################################################################################################################5                                           | 76/99 [00:08<00:02,  8.00it/s] 78%|#################################################################################################################################################4                                         | 77/99 [00:08<00:02,  7.51it/s] 79%|###################################################################################################################################################3                                       | 78/99 [00:08<00:02,  7.25it/s] 80%|#####################################################################################################################################################2                                     | 79/99 [00:08<00:02,  7.00it/s] 82%|#########################################################################################################################################################                                  | 81/99 [00:09<00:02,  8.96it/s] 83%|##########################################################################################################################################################8                                | 82/99 [00:09<00:02,  8.30it/s] 84%|############################################################################################################################################################7                              | 83/99 [00:09<00:02,  7.53it/s] 85%|##############################################################################################################################################################6                            | 84/99 [00:09<00:02,  7.15it/s] 87%|##################################################################################################################################################################4                        | 86/99 [00:09<00:01,  7.36it/s] 88%|####################################################################################################################################################################3                      | 87/99 [00:09<00:01,  6.87it/s] 89%|######################################################################################################################################################################2                    | 88/99 [00:10<00:01,  6.67it/s] 90%|########################################################################################################################################################################1                  | 89/99 [00:10<00:01,  6.42it/s] 92%|###########################################################################################################################################################################8               | 91/99 [00:10<00:00,  8.87it/s] 93%|#############################################################################################################################################################################7             | 92/99 [00:10<00:00,  8.24it/s] 94%|###############################################################################################################################################################################6           | 93/99 [00:10<00:00,  7.21it/s] 95%|#################################################################################################################################################################################5         | 94/99 [00:10<00:00,  6.77it/s] 96%|###################################################################################################################################################################################4       | 95/99 [00:11<00:00,  6.58it/s] 98%|#######################################################################################################################################################################################2   | 97/99 [00:11<00:00,  8.47it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00, 10.33it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  8.69it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s]  4%|#######5                                                                                                                                                                                    | 1/25 [00:00<00:03,  6.20it/s] 12%|######################5                                                                                                                                                                     | 3/25 [00:00<00:03,  7.30it/s] 16%|##############################                                                                                                                                                              | 4/25 [00:00<00:03,  6.94it/s] 20%|#####################################6                                                                                                                                                      | 5/25 [00:00<00:03,  6.55it/s] 28%|####################################################6                                                                                                                                       | 7/25 [00:00<00:02,  8.87it/s] 36%|###################################################################6                                                                                                                        | 9/25 [00:00<00:01, 11.22it/s] 44%|##################################################################################2                                                                                                        | 11/25 [00:01<00:01,  9.53it/s] 52%|#################################################################################################2                                                                                         | 13/25 [00:01<00:01,  8.10it/s] 56%|########################################################################################################7                                                                                  | 14/25 [00:01<00:01,  7.54it/s] 60%|################################################################################################################2                                                                          | 15/25 [00:01<00:01,  7.20it/s] 64%|#######################################################################################################################6                                                                   | 16/25 [00:02<00:01,  6.80it/s] 68%|###############################################################################################################################1                                                           | 17/25 [00:02<00:01,  6.62it/s] 72%|######################################################################################################################################6                                                    | 18/25 [00:02<00:01,  6.50it/s] 80%|#####################################################################################################################################################6                                     | 20/25 [00:02<00:00,  7.05it/s] 88%|####################################################################################################################################################################5                      | 22/25 [00:02<00:00,  7.39it/s] 92%|############################################################################################################################################################################               | 23/25 [00:03<00:00,  6.97it/s] 96%|###################################################################################################################################################################################5       | 24/25 [00:03<00:00,  6.59it/s]100%|###########################################################################################################################################################################################| 25/25 [00:03<00:00,  6.27it/s]100%|###########################################################################################################################################################################################| 25/25 [00:03<00:00,  7.26it/s]
