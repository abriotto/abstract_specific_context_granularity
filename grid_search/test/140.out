# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=128", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.704086184501648, "acc": 0.5061664581298828, "length": 2.7797648906707764, "mode": "train", "epoch": 1}
{"loss": 0.6957604289054871, "acc": 0.48575079441070557, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6933757662773132, "acc": 0.517589807510376, "length": 1.1174590587615967, "mode": "train", "epoch": 2}
{"loss": 0.6926842331886292, "acc": 0.5166382193565369, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.69171541929245, "acc": 0.5281827449798584, "length": 1.1138578653335571, "mode": "train", "epoch": 3}
{"loss": 0.691510796546936, "acc": 0.5190272331237793, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6684337258338928, "acc": 0.5974921584129333, "length": 1.9318697452545166, "mode": "train", "epoch": 4}
{"loss": 0.608620285987854, "acc": 0.712628960609436, "length": 2.6399316787719727, "mode": "test", "epoch": 4}
{"loss": 0.5647402405738831, "acc": 0.7198375463485718, "length": 2.8511791229248047, "mode": "train", "epoch": 5}
{"loss": 0.5734822154045105, "acc": 0.7453652024269104, "length": 3.0682594776153564, "mode": "test", "epoch": 5}
{"loss": 0.5116109251976013, "acc": 0.7589036226272583, "length": 2.9812710285186768, "mode": "train", "epoch": 6}
{"loss": 0.5311917662620544, "acc": 0.7619003653526306, "length": 3.2474403381347656, "mode": "test", "epoch": 6}
{"loss": 0.47696739435195923, "acc": 0.7832140326499939, "length": 3.3037619590759277, "mode": "train", "epoch": 7}
{"loss": 0.5112046599388123, "acc": 0.7652426362037659, "length": 3.4129693508148193, "mode": "test", "epoch": 7}
{"loss": 0.4515983462333679, "acc": 0.7877867221832275, "length": 3.5546395778656006, "mode": "train", "epoch": 8}
{"loss": 0.47898873686790466, "acc": 0.754048764705658, "length": 3.841296911239624, "mode": "test", "epoch": 8}
{"loss": 0.42075997591018677, "acc": 0.8084310293197632, "length": 3.7027084827423096, "mode": "train", "epoch": 9}
{"loss": 0.44830045104026794, "acc": 0.7978833317756653, "length": 3.759385585784912, "mode": "test", "epoch": 9}
{"loss": 0.3837152421474457, "acc": 0.8321605324745178, "length": 3.7225987911224365, "mode": "train", "epoch": 10}
{"loss": 0.4261609613895416, "acc": 0.8197397589683533, "length": 3.873720169067383, "mode": "test", "epoch": 10}
{"loss": 0.3590840995311737, "acc": 0.8433491587638855, "length": 3.853925943374634, "mode": "train", "epoch": 11}
{"loss": 0.39916154742240906, "acc": 0.8412007689476013, "length": 3.865187644958496, "mode": "test", "epoch": 11}
{"loss": 0.33563897013664246, "acc": 0.8574538826942444, "length": 3.8933420181274414, "mode": "train", "epoch": 12}
{"loss": 0.3789884150028229, "acc": 0.8484411239624023, "length": 3.9897611141204834, "mode": "test", "epoch": 12}
{"loss": 0.3163337707519531, "acc": 0.8678173422813416, "length": 3.9553439617156982, "mode": "train", "epoch": 13}
{"loss": 0.31865736842155457, "acc": 0.8720136880874634, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.2990059554576874, "acc": 0.8806403875350952, "length": 3.981820821762085, "mode": "train", "epoch": 14}
{"loss": 0.30353477597236633, "acc": 0.8726109862327576, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.2811521887779236, "acc": 0.8858087062835693, "length": 3.9915332794189453, "mode": "train", "epoch": 15}
{"loss": 0.28231146931648254, "acc": 0.8860920667648315, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.26931607723236084, "acc": 0.8922542929649353, "length": 3.995703935623169, "mode": "train", "epoch": 16}
{"loss": 0.2808745205402374, "acc": 0.8896756172180176, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.2566715478897095, "acc": 0.8980090022087097, "length": 3.996532440185547, "mode": "train", "epoch": 17}
{"loss": 0.2627306878566742, "acc": 0.8971843123435974, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.24151410162448883, "acc": 0.904776930809021, "length": 3.997771739959717, "mode": "train", "epoch": 18}
{"loss": 0.2528422772884369, "acc": 0.9048633575439453, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.2322424054145813, "acc": 0.910365104675293, "length": 3.99877667427063, "mode": "train", "epoch": 19}
{"loss": 0.24210381507873535, "acc": 0.9070819020271301, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.23397807776927948, "acc": 0.9074660539627075, "length": 3.9990196228027344, "mode": "train", "epoch": 20}
{"loss": 0.24720178544521332, "acc": 0.903839647769928, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.22034268081188202, "acc": 0.9152317047119141, "length": 3.999561309814453, "mode": "train", "epoch": 21}
{"loss": 0.2388630360364914, "acc": 0.9134812951087952, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.19956240057945251, "acc": 0.9260032176971436, "length": 3.999363422393799, "mode": "train", "epoch": 22}
{"loss": 0.2246202975511551, "acc": 0.9185152649879456, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.186991348862648, "acc": 0.9309174418449402, "length": 3.9997305870056152, "mode": "train", "epoch": 23}
{"loss": 0.18870943784713745, "acc": 0.931399405002594, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.17719873785972595, "acc": 0.9349035620689392, "length": 3.9995338916778564, "mode": "train", "epoch": 24}
{"loss": 0.21015648543834686, "acc": 0.9238054156303406, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.17573651671409607, "acc": 0.9349206686019897, "length": 3.999373197555542, "mode": "train", "epoch": 25}
{"loss": 0.20031030476093292, "acc": 0.9275597333908081, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.1783657819032669, "acc": 0.9348533153533936, "length": 3.999786853790283, "mode": "train", "epoch": 26}
{"loss": 0.19623902440071106, "acc": 0.9372013807296753, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.16049166023731232, "acc": 0.9437913298606873, "length": 3.9993972778320312, "mode": "train", "epoch": 27}
{"loss": 0.1742185801267624, "acc": 0.9445392489433289, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.1492929905653, "acc": 0.9487432241439819, "length": 3.999953269958496, "mode": "train", "epoch": 28}
{"loss": 0.1742352992296219, "acc": 0.9415528178215027, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.1410670429468155, "acc": 0.9519634246826172, "length": 3.999945640563965, "mode": "train", "epoch": 29}
{"loss": 0.16164904832839966, "acc": 0.947781503200531, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.13961273431777954, "acc": 0.9529909491539001, "length": 3.9999589920043945, "mode": "train", "epoch": 30}
{"loss": 0.16375258564949036, "acc": 0.9449657797813416, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.1332399845123291, "acc": 0.956009566783905, "length": 3.999903678894043, "mode": "train", "epoch": 31}
{"loss": 0.16070397198200226, "acc": 0.9466721415519714, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.12964341044425964, "acc": 0.9562975764274597, "length": 3.999955654144287, "mode": "train", "epoch": 32}
{"loss": 0.15483759343624115, "acc": 0.9543513059616089, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.11849042028188705, "acc": 0.9604815244674683, "length": 3.999890089035034, "mode": "train", "epoch": 33}
{"loss": 0.16592322289943695, "acc": 0.9515359997749329, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.12112436443567276, "acc": 0.9592070579528809, "length": 3.999994993209839, "mode": "train", "epoch": 34}
{"loss": 0.14074240624904633, "acc": 0.955716609954834, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.12399271130561829, "acc": 0.9551276564598083, "length": 3.9999589920043945, "mode": "train", "epoch": 35}
{"loss": 0.1559172421693802, "acc": 0.9542662501335144, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.12713192403316498, "acc": 0.9603760838508606, "length": 3.999994993209839, "mode": "train", "epoch": 36}
{"loss": 0.1646299809217453, "acc": 0.9499146938323975, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.12061017751693726, "acc": 0.9615735411643982, "length": 3.9999990463256836, "mode": "train", "epoch": 37}
{"loss": 0.13480189442634583, "acc": 0.9567405581474304, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.11129014194011688, "acc": 0.964823305606842, "length": 3.9999990463256836, "mode": "train", "epoch": 38}
{"loss": 0.15852133929729462, "acc": 0.9546076655387878, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.12146709859371185, "acc": 0.9595493674278259, "length": 3.999999523162842, "mode": "train", "epoch": 39}
{"loss": 0.14615526795387268, "acc": 0.9502559304237366, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.12114940583705902, "acc": 0.9589508771896362, "length": 3.9999988079071045, "mode": "train", "epoch": 40}
{"loss": 0.18567290902137756, "acc": 0.945733904838562, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.1443701684474945, "acc": 0.9513396620750427, "length": 3.9999990463256836, "mode": "train", "epoch": 41}
{"loss": 0.16210494935512543, "acc": 0.9505120515823364, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.12520286440849304, "acc": 0.9609748125076294, "length": 3.9999990463256836, "mode": "train", "epoch": 42}
{"loss": 0.15229423344135284, "acc": 0.953924834728241, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.11288394778966904, "acc": 0.9641106128692627, "length": 4.0, "mode": "train", "epoch": 43}
{"loss": 0.1510942131280899, "acc": 0.9598978161811829, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.11137041449546814, "acc": 0.9639110565185547, "length": 3.9999983310699463, "mode": "train", "epoch": 44}
{"loss": 0.13825328648090363, "acc": 0.9574233889579773, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.10626760125160217, "acc": 0.9654503464698792, "length": 3.999999523162842, "mode": "train", "epoch": 45}
{"loss": 0.11925184726715088, "acc": 0.9657850861549377, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.10672341287136078, "acc": 0.9641961455345154, "length": 4.0, "mode": "train", "epoch": 46}
{"loss": 0.1464615911245346, "acc": 0.9553754925727844, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.10605686157941818, "acc": 0.9657638669013977, "length": 3.999997138977051, "mode": "train", "epoch": 47}
{"loss": 0.15079070627689362, "acc": 0.9570819735527039, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.10749844461679459, "acc": 0.9639396071434021, "length": 4.0, "mode": "train", "epoch": 48}
{"loss": 0.1513489931821823, "acc": 0.9483787417411804, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.11585488170385361, "acc": 0.9614595770835876, "length": 4.0, "mode": "train", "epoch": 49}
{"loss": 0.15360116958618164, "acc": 0.9543514251708984, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.11235469579696655, "acc": 0.9634547829627991, "length": 4.0, "mode": "train", "epoch": 50}
{"loss": 0.1356266587972641, "acc": 0.9577645063400269, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.10963364690542221, "acc": 0.9657639861106873, "length": 4.0, "mode": "train", "epoch": 51}
{"loss": 0.13074564933776855, "acc": 0.959214985370636, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.12419237196445465, "acc": 0.9613168835639954, "length": 4.0, "mode": "train", "epoch": 52}
{"loss": 0.15886935591697693, "acc": 0.946331262588501, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.1307181417942047, "acc": 0.9582383036613464, "length": 3.999997854232788, "mode": "train", "epoch": 53}
{"loss": 0.15910065174102783, "acc": 0.9491468071937561, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.11910296976566315, "acc": 0.9612028002738953, "length": 3.9999988079071045, "mode": "train", "epoch": 54}
{"loss": 0.13916970789432526, "acc": 0.9552049040794373, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.1155136227607727, "acc": 0.9630557298660278, "length": 4.0, "mode": "train", "epoch": 55}
{"loss": 0.16208986937999725, "acc": 0.9462458491325378, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.11977168917655945, "acc": 0.9601196050643921, "length": 4.0, "mode": "train", "epoch": 56}
{"loss": 0.1545395404100418, "acc": 0.9546929597854614, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.09803169220685959, "acc": 0.9678730964660645, "length": 3.999995470046997, "mode": "train", "epoch": 57}
{"loss": 0.12966924905776978, "acc": 0.964590311050415, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.08469174057245255, "acc": 0.9742589592933655, "length": 4.0, "mode": "train", "epoch": 58}
{"loss": 0.11389394104480743, "acc": 0.9658704400062561, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.08415700495243073, "acc": 0.9743443131446838, "length": 4.0, "mode": "train", "epoch": 59}
{"loss": 0.11283200979232788, "acc": 0.9622015357017517, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.07601691782474518, "acc": 0.9776511192321777, "length": 4.0, "mode": "train", "epoch": 60}
{"loss": 0.0929737314581871, "acc": 0.9733787775039673, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.0721367821097374, "acc": 0.9796750545501709, "length": 4.0, "mode": "train", "epoch": 61}
{"loss": 0.09768607467412949, "acc": 0.974146842956543, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.0762462466955185, "acc": 0.9782212376594543, "length": 4.0, "mode": "train", "epoch": 62}
{"loss": 0.09185513108968735, "acc": 0.974232017993927, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.06758484244346619, "acc": 0.9813283085823059, "length": 4.0, "mode": "train", "epoch": 63}
{"loss": 0.11102145165205002, "acc": 0.968686044216156, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.06464025378227234, "acc": 0.9820980429649353, "length": 4.0, "mode": "train", "epoch": 64}
{"loss": 0.0982770025730133, "acc": 0.9732936024665833, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.06325826048851013, "acc": 0.9820696711540222, "length": 4.0, "mode": "train", "epoch": 65}
{"loss": 0.08916319906711578, "acc": 0.9744027256965637, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.060448452830314636, "acc": 0.9834378957748413, "length": 4.0, "mode": "train", "epoch": 66}
{"loss": 0.09084740281105042, "acc": 0.974317193031311, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.05826190859079361, "acc": 0.9827821850776672, "length": 4.0, "mode": "train", "epoch": 67}
{"loss": 0.07986216992139816, "acc": 0.9770477414131165, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.07019704580307007, "acc": 0.9809293746948242, "length": 4.0, "mode": "train", "epoch": 68}
{"loss": 0.089272640645504, "acc": 0.9739760160446167, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.06200817972421646, "acc": 0.9824399948120117, "length": 3.999999761581421, "mode": "train", "epoch": 69}
{"loss": 0.10746172815561295, "acc": 0.9727818369865417, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.07121288031339645, "acc": 0.9805017113685608, "length": 4.0, "mode": "train", "epoch": 70}
{"loss": 0.11216466873884201, "acc": 0.9717578291893005, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.0704115554690361, "acc": 0.9791334271430969, "length": 4.0, "mode": "train", "epoch": 71}
{"loss": 0.10777020454406738, "acc": 0.9704779386520386, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.06307664513587952, "acc": 0.9815849661827087, "length": 4.0, "mode": "train", "epoch": 72}
{"loss": 0.12662072479724884, "acc": 0.9693686962127686, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.05698616802692413, "acc": 0.983466386795044, "length": 4.0, "mode": "train", "epoch": 73}
{"loss": 0.10311730206012726, "acc": 0.9715871810913086, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.06702234596014023, "acc": 0.9807868599891663, "length": 4.0, "mode": "train", "epoch": 74}
{"loss": 0.0924784392118454, "acc": 0.973549485206604, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.057210516184568405, "acc": 0.9842644333839417, "length": 3.9999988079071045, "mode": "train", "epoch": 75}
{"loss": 0.0937340185046196, "acc": 0.9745732545852661, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.068948894739151, "acc": 0.9790764451026917, "length": 4.0, "mode": "train", "epoch": 76}
{"loss": 0.12312660366296768, "acc": 0.9696246981620789, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.06011898070573807, "acc": 0.9828677177429199, "length": 4.0, "mode": "train", "epoch": 77}
{"loss": 0.09921324253082275, "acc": 0.97380530834198, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.05281618982553482, "acc": 0.9849485754966736, "length": 4.0, "mode": "train", "epoch": 78}
{"loss": 0.1044440045952797, "acc": 0.97457355260849, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.047645095735788345, "acc": 0.9868586659431458, "length": 4.0, "mode": "train", "epoch": 79}
{"loss": 0.08871199935674667, "acc": 0.9758530855178833, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.04786669835448265, "acc": 0.9866018891334534, "length": 4.0, "mode": "train", "epoch": 80}
{"loss": 0.0851481482386589, "acc": 0.9792661666870117, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.044543106108903885, "acc": 0.9877423048019409, "length": 4.0, "mode": "train", "epoch": 81}
{"loss": 0.08538662642240524, "acc": 0.9794368743896484, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.047657422721385956, "acc": 0.9861743450164795, "length": 4.0, "mode": "train", "epoch": 82}
{"loss": 0.094809390604496, "acc": 0.9758532047271729, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.05763827636837959, "acc": 0.9832383394241333, "length": 4.0, "mode": "train", "epoch": 83}
{"loss": 0.12089967727661133, "acc": 0.9702219367027283, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.07382671535015106, "acc": 0.9773660898208618, "length": 4.0, "mode": "train", "epoch": 84}
{"loss": 0.14312876760959625, "acc": 0.9635664820671082, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.09433488547801971, "acc": 0.9696978330612183, "length": 4.0, "mode": "train", "epoch": 85}
{"loss": 0.12530331313610077, "acc": 0.9648463726043701, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.0939423143863678, "acc": 0.96778804063797, "length": 4.0, "mode": "train", "epoch": 86}
{"loss": 0.13914279639720917, "acc": 0.9625426530838013, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.07090359181165695, "acc": 0.979930579662323, "length": 3.999990463256836, "mode": "train", "epoch": 87}
{"loss": 0.11599712818861008, "acc": 0.972440242767334, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.050948720425367355, "acc": 0.9842076897621155, "length": 4.0, "mode": "train", "epoch": 88}
{"loss": 0.09635412693023682, "acc": 0.973976194858551, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.04569320008158684, "acc": 0.9860868453979492, "length": 3.999979019165039, "mode": "train", "epoch": 89}
{"loss": 0.0926641896367073, "acc": 0.9764505624771118, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.05587409809231758, "acc": 0.9830387830734253, "length": 3.999999761581421, "mode": "train", "epoch": 90}
{"loss": 0.10328560322523117, "acc": 0.9756826758384705, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.05661943927407265, "acc": 0.98386549949646, "length": 3.999999761581421, "mode": "train", "epoch": 91}
{"loss": 0.12395455688238144, "acc": 0.9680032134056091, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.049938250333070755, "acc": 0.9853191375732422, "length": 4.0, "mode": "train", "epoch": 92}
{"loss": 0.10028091818094254, "acc": 0.9744879007339478, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.06458188593387604, "acc": 0.980701208114624, "length": 4.0, "mode": "train", "epoch": 93}
{"loss": 0.11057287454605103, "acc": 0.9730373620986938, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.04968242719769478, "acc": 0.9864025712013245, "length": 4.0, "mode": "train", "epoch": 94}
{"loss": 0.09038253128528595, "acc": 0.9772182703018188, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.047714028507471085, "acc": 0.9870011806488037, "length": 4.0, "mode": "train", "epoch": 95}
{"loss": 0.09080932289361954, "acc": 0.9799489378929138, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.04296885058283806, "acc": 0.988312304019928, "length": 4.0, "mode": "train", "epoch": 96}
{"loss": 0.09098855406045914, "acc": 0.9781569242477417, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.04601630941033363, "acc": 0.9859150648117065, "length": 3.999972105026245, "mode": "train", "epoch": 97}
{"loss": 0.08173204958438873, "acc": 0.9782420992851257, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.04303765669465065, "acc": 0.9872007369995117, "length": 4.0, "mode": "train", "epoch": 98}
{"loss": 0.09468436986207962, "acc": 0.9769623279571533, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.038634490221738815, "acc": 0.9889965653419495, "length": 4.0, "mode": "train", "epoch": 99}
{"loss": 0.08616309612989426, "acc": 0.9793514609336853, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.039614658802747726, "acc": 0.9888826012611389, "length": 4.0, "mode": "train", "epoch": 100}
{"loss": 0.0933595821261406, "acc": 0.9764505624771118, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.7454546093940735
